### DISCUSSION TITLE
How could I change the "price" for energy?

### DISCUSSION THREAD

**Discussion by Kreatinbulle** [hif2k1/battery_sim#73] [Q&A] [answered]
Hi, I set up the intregration some days ago and it is running flawlessly. Now I have a price change for my imported energy and I want to adopt this. I did not set up the sensors with an yaml but used the default method.

**Comment by hif2k1** [comment, marked as answer]
Sorry, didn't see you had started this discussion. There is a feature request for this (https://github.com/hif2k1/battery_sim/issues/65). I will try to implement this in a future release. In the mean time you can edit the configuration by editing the file core.configentries which is in the hidden .storage directory in your home assistant config directory.
